datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/deser/key/ZonedDateTimeKeyDeserializer.java

```diff
@@ -17,9 +16,8 @@ private ZonedDateTimeKeyDeserializer() {
 
     @Override
     protected ZonedDateTime deserialize(String key, DeserializationContext ctxt) throws IOException {
-        // not serializing timezone data yet
         try {
-            return ZonedDateTime.parse(key, DateTimeFormatter.ISO_OFFSET_DATE_TIME);
+            return ZonedDateTime.parse(key);
```

Comment:
Could you add a comment stating that no formatter passed to allow use of all supported formats?

Reply:
Done